webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -333,6 +361,24 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     };
   }, []);
 
+  const updateExpListView = useCallback(
+    (view: ExpListView) => {
+      if (view === 'paged') {
+        setPage(Math.max(Math.round(((page - 1) * PAGE_SIZE) / settings.pageLimit), 1));
```

Comment:
fix: I'm not sure what's causing it, but when I scroll all the way to the bottom of the Scroll View and then switch to Paged, I end up going to page 5/11 instead of to the end like I'd expect

nit: use page like this instead: `setPage((prevPage) => ... )`

Reply:
I tried to fix it a little bit, but since `PAGE_SIZE` is 100 and  `settings.pageLimit` can be 20, and we do not know where did the scroll stoped within that 100 rows, so the page calculation may not be very accurate